### PR TITLE
Add snippets list command

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: Services/SnippetsReader.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        $inputAdapter = new \Enlight_Config_Adapter_File([
+            'configDir' => $snippetsDir,
+        ]);
+
+        $finder = new Finder();
+        $finder->files()->in($snippetsDir);
+
+        foreach ($finder as $file) {
+            $filePath = $file->getRelativePathname();
+            if (strpos($filePath, '.ini') == strlen($filePath) - 4) {
```
Comment (Jnoack331) [inline Services/SnippetsReader.php]: why not `$file->getExtension() === "ini"`

Comment (sd-lueckel) [reply Services/SnippetsReader.php]: done!

File: Services/SnippetsReader.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        $inputAdapter = new \Enlight_Config_Adapter_File([
+            'configDir' => $snippetsDir,
+        ]);
+
+        $finder = new Finder();
+        $finder->files()->in($snippetsDir);
+
+        foreach ($finder as $file) {
+            $filePath = $file->getRelativePathname();
+            if ('.ini' === $file->getExtension()) {
```
Comment (Jnoack331) [inline Services/SnippetsReader.php]: The dot must be removed (http://php.net/manual/de/splfileinfo.getextension.php)

**Review by Jnoack331 — CHANGES_REQUESTED** [review]

**Review by Jnoack331 — APPROVED** [review]

File: Services/SnippetsReader.php (hunk 2 above)
Comment (sd-lueckel) [reply Services/SnippetsReader.php]: done!

**Review by twojtylak — APPROVED** [review]

**Pull request merged**